src/main/java/com/fasterxml/jackson/databind/deser/BeanDeserializerFactory.java

```diff
@@ -661,6 +664,22 @@ protected void addBeanProps(DeserializationContext ctxt,
         }
     }
 
+    // since 2.18
+    private AnnotatedMember _findCreatorPropWithAnySetter(DeserializationContext ctxt, SettableBeanProperty[] creatorProps) {
+        if (creatorProps != null) {
+            AnnotationIntrospector ai = ctxt.getAnnotationIntrospector();
```

Comment:
I think we should not need to ask `AnnotationIntrospector` again, but see if we have `CreatorProperty`, with flag set.